libs/utils/utils.js

```diff
@@ -731,6 +731,8 @@ function decorateDefaults(el) {
 }
 
 function decorateHeader() {
+  const breadcrumbs = document.querySelector('.breadcrumbs');
```

Comment:
I noticed that if a breadcrumbs is authored as first block, we immediately load it. The global-navigation should take care of this and on L752 we move the breadcrumbs into the header `if (breadcrumbs) header.append(breadcrumbs);`

Reply:
// FYI @salonijain3 @sharmrj @bandana147

Reply:
@mokimo where are you seeing that we immediately load this? For me it looks like it was already working properly before any change here. 😅

Reply:
Here's a test experience @vhargrave https://main--milo--adobecom.aem.page/drafts/osahin/education

When you turn the header off, it treats the breadcrumbs as block, which 404s.

Reply:
ah ok, i see thanks 👍